src/components/input/Input.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import toJSON from 'enzyme-to-json'
+import { shallow, mount } from 'enzyme'
+
+import Input from './Input'
+
+describe('<Input/>', () => {
+  it('matches snapshot', () => {
+    const id = 'InputId'
+    const label = 'Input field'
+    const type = 'text'
+    const required = true
+    const component = shallow(
+      <Input id={id} label={label} type={type} required={required} />
+    )
+    expect(toJSON(component)).toMatchSnapshot()
+  })
+
+  it('should invoke onChange when triggered', () => {
+    const onChangeSpy = jest.fn()
+    const component = mount(<Input onChange={onChangeSpy} />)
+    component.find('input').simulate('change')
+    expect(onChangeSpy).toHaveBeenCalledTimes(1)
+  })
+
+  it('should change label focused prop on focus and blur of input', () => {
+    const id = 'InputId'
+    const label = 'Input field'
+    const type = 'text'
+    const component = shallow(
+      <Input label={label} type={type} id={id} required />
+    )
```

Comment:
These props are unnecessary since their presence do not impact the test case. They can be removed 🙂

Reply:
💯 Removed!